Spanner/src/Batch/BatchSnapshot.php

```diff
@@ -212,16 +212,20 @@ public function partitionQuery($sql, array $options = [])
      * ```
      *
      * @param PartitionInterface $partition The partition to read.
-     * @param array $options Configuration Options.
+     * @param array $options {
+     *     Additional configuration Options.
```

Comment:
I think I'll just reverse the order on lines 275 and 289, this will be less confusing.

Technically we could apply this on `BatchClient::snapshot()`. I think in this case we would also need to update serialization logic for `BatchSnapshot`, `ReadPartition`, and `QueryPartition` to handle `requestTag` for consistency. Do we need that?